package.json

```diff
@@ -10,6 +10,7 @@
     "@types/node": "^12.0.0",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
+    "node-sass": "^5.0.0",
```

Comment:
Not using ^6.0.0 to avoid `Node Sass version 6.0.1 is incompatible with ^4.0.0 || ^5.0.0.` error